config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -252,3 +252,63 @@ periodics:
     - name: creds
       secret:
         secretName: resultstore-service-account
+- cron: "54 * * * *"  # Every hour at 54 minutes past the hour
+  name: ci-test-infra-branchprotector
+  cluster: test-infra-trusted
+  labels:
+    app: branchprotector
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+  spec:
+    containers:
+    - name: branchprotector
+      image: gcr.io/k8s-prow/branchprotector:v20190314-e8134a3
+      args:
+      - --config-path=prow/config.yaml
+      - --job-config-path=config
+      - --github-token-path=/etc/github/oauth
+      - --confirm
+      - --github-endpoint=http://ghproxy.default.svc.cluster.local
+      - --github-endpoint=https://api.github.com
+      volumeMounts:
+      - name: oauth
+        mountPath: /etc/github
+        readOnly: true
+    restartPolicy: Never
```

Comment:
do we need `restartPolicy` anymore? since the lifecycle will be controlled by prow now

Reply:
Nope, we don't need it any more. Looks like Prow forces this value when creating pods for PJs: https://github.com/kubernetes/test-infra/blob/fbc764e23acba94e57ebb15812dbebbbc6a9050b/prow/pod-utils/decorate/podspec.go#L156
I'll remove this.